cppcache/integration-test-2/PdxInstanceTest.cpp

```diff
@@ -204,10 +204,10 @@ TEST(PdxInstanceTest, testPdxInstance) {
   EXPECT_EQ(1, cachePerfStats.getPdxInstanceDeserializations())
       << "pdxInstanceDeserialization should be equal to 1.";
 
-  EXPECT_EQ(0, cachePerfStats.getPdxInstanceCreations())
+  EXPECT_TRUE(cachePerfStats.getPdxInstanceCreations() == 0)
```

Comment:
Why is this changed? The `EXPECT_EQ` is more readable and provides a better error message.